### ISSUE TITLE
Test tutorials for 18.3 release candidate

### ISSUE THREAD

**Issue by sbailey** [desihub/tutorials#13]
When we have a release candidate for software release 18.3, test tutorials with that set of tags and make updates as necessary (to tutorials or code) prior to making final set of tags for 18.3.  In all cases they should be updated to work with jupyter-dev.nersc.gov with code installed at NERSC.

  * [x] Intro_to_DESI_spectra
  * [x] dc17a-truth (consider renaming / making more general)
  * [x] quickgen (deprecate / remove?)
  * [x] simulating_desi_spectra
  * [x] survey-simulations
  * [x] redrock/RedrockOutputs
  * [x] redrock/RedrockBOSSDemo
  * [x] redrock/rrplotspec

**Comment by sbailey** [comment]
Update:

redrock tutorials are still broken since `redrock.io.read_templates` was removed; I haven't found the convenient interactive workaround for that functionality yet.

I archived quickgen tutorial into a new archive directory (it has been superseded by simulating_desi_spectra tutorial).  I checked that the others work with the 18.3 kernel at NERSC and made minor updates as needed.

Leaving this open until the redrock tutorials are fixed.
